src/styles/app.styl

```diff
@@ -29,6 +29,21 @@ body
     display flex
     min-width 100%
     width 100%
+    background-color var(--silver)
+    background-repeat no-repeat
+    background-size cover
+    background-position center
+    mix-blend-mode multiply
+
+.App::before
+    content ''
+    position absolute
+    top 0
+    left 0
+    width 100%
+    height 100%
+    background-color rgba(0, 0, 0, .32)
```

Comment:
Is it the `divider`?

Reply:
No, this is the "black" filter in front of the background

Reply:
dans tous les thèmes ?

Reply:
Le wallpaper n'a pas été défini pour différents thèmes

Reply:
Je confirme, le wallpaper ne change et le "black" filter ne sont pas sensés changer en fonction du thème cc @joel-costa

Reply:
nit: peut-être rajouter un petit commentaire dans le stylus pour l'expliquer